### PR TITLE
solution for 5-CORE-View-an-individual-article

### PR DESCRIPTION
**Pull request by miors**
added individual article page
added routing associated functionalities including BrowserRoutes, Routes, Route and Navigate
Hide button when individual article page is shown

### CODE REVIEW

**Review by pippaamy — COMMENTED** [review]
Great job - I would add the loading in :)

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+import IndArticlePage from "./components/IndArticlePage";
+import Article from "./components/Article";
+import { Routes, Route } from "react-router-dom";
+import api from "./api/api";
 
 function App() {
-  const [count, setCount] = useState(0);
+  const [articlesList, setArticlesList] = useState([]);
+
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (miors) [reply src/App.jsx:14]: Thank you. I have moved it back to Articles list page :)

File: src/components/IndArticlePage.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { useParams } from "react-router-dom";
+import { useState } from "react";
+import api from "../api/api";
+import { useEffect } from "react";
+import Article from "./Article";
+
+export default function IndArticlePage() {
+  const [article, setArticle] = useState({});
```
[the remark this answers is not in the document]
Comment (miors) [reply src/components/IndArticlePage.jsx:8]: Done

File: src/components/AllArticlesPage.jsx [hunk 3]
```diff
@@ -2,15 +2,7 @@ import { useState, useEffect } from "react";
 import api from "../api/api";
 import Article from "./Article";
```
[the remark this answers is not in the document]
Comment (miors) [reply src/components/AllArticlesPage.jsx:4]: Done

**Pull request merged**